src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1010,6 +1010,9 @@ bool RaftPart::prepareElectionRequest(
         return false;
     }
 
+    // Before start a new election, reset the votedAddr
+    votedAddr_ = std::make_pair(0, 0);
```

Comment:
why we do not vote myself? In the raft paper, we should vote for myself first.

Reply:
Maybe the variable name is not clear enough. It is just uesed to recored whether current replica has voted for somebody else in the requested term.